cmd/server.go

```diff
@@ -68,6 +68,7 @@ const (
 	RepoWhitelistFlag          = "repo-whitelist"
 	RequireApprovalFlag        = "require-approval"
 	RequireMergeableFlag       = "require-mergeable"
+	SilenceForkPRErrorsFlag    = "silence-fork-pr-errors"
```

Comment:
looks great! just need to update the tests in `server_test.go` look for each instance of the whitelist flag and then put your flag in there too

Reply:
Unless I'm missing it, it doesn't look like `SilenceWhitelistErrorsFlag` is test in `server_test.go`. I can add tests for both Silence flags if you would like.